package.json

```diff
@@ -22,21 +26,23 @@
       "html"
     ]
   },
+  "browserify": {
```

Comment:
Changes should be minimal.
No need to move section to another place in the file